src/webchat-ui/components/presentational/Header.tsx

```diff
@@ -91,6 +107,10 @@ const Header: FC<HeaderProps> = props => {
 		chatToggleButtonRef?.current?.focus?.();
 	};
 
+	const handleMenuClick = () => {
+		onSetShowChatOptionsScreen?.(true);
+	};
+
 	const isCompact =
```

Comment:
You may want to test this with 
```javascript
headerLogoUrl: `https://www.cognigy.com/hubfs/logo_cognigy%20(597x79)SVG.svg`,
```
config setting.

Currently the header with options remains expanded if we have this setting enabled. It should always be "compact" in this screen according to Figma designs.

Reply:
When I tried with logo url, the chat options header looked 'compact', just as in the prototype.

Reply:
I found an issue with the header loyout. It is a pre-existing issue, but is very visible after introducing the chat-options screen.  I have created a ticket in our backlog and the ticket number is #59096